pyvista/core/utilities/points.py

```diff
@@ -228,12 +234,65 @@ def fit_plane_to_points(points, return_meta=False):
     ... )
     >>> pl.show()
 
+    Fit a plane to a mesh.
+
+    >>> import pyvista
+    >>> from pyvista import examples
+    >>>
+    >>> # Create mesh
+    >>> mesh = examples.download_shark()
+    >>>
+    >>> # Fit plane
+    >>> plane = pyvista.fit_plane_to_points(mesh.points)
+    >>>
+    >>> # Plot the fitted plane
+    >>> pl = pyvista.Plotter()
+    >>> _ = pl.add_mesh(
+    ...     plane, show_edges=True, color='lightblue', opacity=0.25
+    ... )
+    >>> _ = pl.add_mesh(mesh, color='gray')
+    >>> pl.camera_position = [
+    ...     (-117, 76, 235),
+    ...     (1.69, -1.38, 0),
+    ...     (0.189, 0.957, -0.22),
+    ... ]
+    >>> pl.show()
+
     """
+    # Apply SVD to get orthogonal basis vectors to define the plane
     data = np.array(points)
-    center = data.mean(axis=0)
-    result = np.linalg.svd(data - center)
-    normal = np.cross(result[2][0], result[2][1])
-    plane = pyvista.Plane(center=center, direction=normal)
+    data_center = data.mean(axis=0)
+    U, S, Vh = np.linalg.svd(data - data_center)
```

Comment:
Let's remove unused variables.
```suggestion
    _, _, Vh = np.linalg.svd(data - data_center)
```